src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -439,6 +459,7 @@ export default {
 		const scroller = ref(null)
 
 		const showArchived = ref(false)
+		const filters = ref([])
```

Comment:
Instead of doing this in `mounted`? I don't see why we should call `handleFilter()`
```suggestion
		const filters = ref(BrowserStorage.getItem('filterEnabled')?.split(',') ?? [])
```